paris-processor/src/main/java/com/airbnb/paris/processor/ParisProcessor.kt

```diff
@@ -110,30 +113,30 @@ class ParisProcessor : SkyProcessor(), WithParisProcessor {
         externalStyleablesInfo = BaseStyleableInfoExtractor(this).fromEnvironment()
 
         val allStyleables = styleablesInfo + externalStyleablesInfo
+        if (allStyleables.isNotEmpty() && rFinder.element == null) {
```

Comment:
If there is a null r class write the style appliers code fail, so moving this earlier